winlogbeat/sys/strings_windows.go

```diff
@@ -0,0 +1,48 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package sys
+
+import (
+	"sync"
+
+	"golang.org/x/sys/windows"
+	"golang.org/x/text/encoding"
+	"golang.org/x/text/encoding/charmap"
+)
+
+func initANSIDecoder() *encoding.Decoder {
+	ansiCP := windows.GetACP()
+	for _, enc := range charmap.All {
+		cm, ok := enc.(*charmap.Charmap)
+		if !ok {
+			continue
+		}
+		cmID, _ := cm.ID()
+		if uint32(cmID) != ansiCP {
+			continue
+		}
+		return cm.NewDecoder()
+	}
+	return charmap.Windows1250.NewDecoder()
```

Comment:
Can you put a comment here about why Windows-1250 is here. I figured windows-1252 would be used as it is was the default, but I'm in the Americas.

Reply:
I thought about that since there is no good default to appeal to every system. A config option could be a solution to override the ACP call in case it is not found, but seems a bit non intuitive to have a config to fallback onto. Maybe windows-1252 is used by a larger number of customers, and we can add a debug log to note the default is used. WDYT?

Reply:
My naive assumption is that this will very rarely be executed so I don't think we need to add any special handling. I would just ask that we put comment here about our assumptions and also to identify 1250 as something like 'Central European' for those not familiar with code pages.